src/plugins/overlay/_base.scss

```diff
@@ -78,6 +80,16 @@
 	width: 90%;
 }
 
+.wb-popup-br {
+	@extend %overlay-popup;
+	border-radius: 6px;
+	bottom: 20px;
+	font-size: .9em;
```

Comment:
Please remove this `font-size` property, it shouldn't belong to the overlay itself, but instead to whatever goes in the overlay if font size is needed to be different i.e. chat wizard in GCWeb.